### ISSUE TITLE
Support v2

### ISSUE THREAD

**Issue by pavel-rty** [tolgee/php-example#1]
Are there any plans to support v2 API for PHP?

**Comment by JanCizmar** [comment]
Hello! What do you mean by v2 API?


If you mean our REST API v2... You can interact with it using by calling the endpoints using [curl](https://www.php.net/manual/en/book.curl.php).

**Comment by pavel-rty** [comment]
Yes. Correct. So, you mean we can't use this repo and this CDK in PHP project? Only custom code?

**Comment by JanCizmar** [comment]
Unfortunately, yes.
